index.js

```diff
@@ -141,7 +142,7 @@ function launchSpec(kernelSpec, spawnOptions) {
  * @return {string}       spawnResults.connectionFile  connection file path
  * @return {object}       spawnResults.config          connectionConfig
  */
-function launch(kernelName, spawnOptions, specs) {
+function launch(kernelName, spawnOptions = {}, specs) {
```

Comment:
We'll need babel config for the default assignment here. Fun fact though:

```
> Object.assign({}, {a: 2}, undefined)
{ a: 2 }
```

It's ok if `spawnOptions` is `undefined` for the `Object.assign`.